handlers/deploy.go

```diff
@@ -31,6 +31,9 @@ const watchdogPort = 8080
 // initialReplicasCount how many replicas to start of creating for a function
 const initialReplicasCount = 1
 
+// nonRootFunctionuserID is the user id that is set when DeployHandlerConfig.ForceNonRootUser is true
+const nonRootFunctionuserID = 2000
```

Comment:
@stefanprodan mentioned using > 10000 according to "kubesec" advisories

Reply:
https://kubesec.io/basics/containers-securitycontext-runasuser/

Reply:
Yes that's the one. Wdyt?

Reply:
updating it right now to `12000`